server/service/IndividualServicesService.js

```diff
@@ -193,9 +193,9 @@ exports.bequeathYourDataAndDie = function (body, user, originator, xCorrelator,
         
         let serverAddress = await tcpServerInterface.getLocalAddressOfTheProtocol(applicationProtocol);
```

Comment:
We can remove the codeblock from line 194 to 199 as this will violate the idempotence concept